packages/waas/src/networks.ts

```diff
@@ -0,0 +1,60 @@
+
+const RPC_BASE = 'https://nodes.sequence.app/'
+
+const nameToId = {
+  'mainnet': 1,
```

Comment:
we should be using `@0xsequence/network` instead, it exports a `networks` object which is our canonical network list of names and ids

Reply:
@marino39 Can you fix this? As Peter mentioned above, `networks` package offers a really nice interface for this already and there is no need to hardcode these values here. Let's standardize.

Reply:
Yes, I have missed that one.